tools/iree-prof-output-stdout.cc

```diff
@@ -34,6 +34,18 @@ const char* ArchToString(tracy::CpuArchitecture arch) {
   }
 }
 
+std::string MemToString(double mem_usage) {
+  if (mem_usage > 1000 * 1000 * 1000) {
```

Comment:
I think we can use `>=`.

Reply:
Also, since there is a round up logic, we may need to compare after the rounding.

Reply:
Done